specs/DisableCrashReporting.md

```diff
@@ -0,0 +1,126 @@
+Process Info
+===
+
+# Background
+Currently, when a WebView2 is created a WebView2 crashpad process will be created in addition to starting a WebView2 browser process. If any WebView2 process crashes, the crashpad process will create crash dumps and send them to Microsoft for diagnosis. This document covers new APIs to allow the end developer to customize crash reporting to help when running diagnostics and doing analysis. They can set the `CoreWebView2EnvironmentOptions.IsCustomCrashReportingEnabled` property to not have crash dumps sent to Microsoft and use the `CoreWebView2Environment.CrashDumpFolderPath` property to locate crash dumps and do customization with them instead.
+
+# Examples
+## WinRT and .NET   
+```c#
+
+/// Create WebView Environment with option
+void CreateEnvrionmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.IsCustomCrashReportingEnabled = true;
+    CoreWebView2Environment environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+}
+
+string _crashDumpFolder;
+
+void GetDumpFolder()
+{
+    _crashDumpFolder = webView.CoreWebView2.Environment.CrashDumpFolderPath;
```

Comment:
We probably don't need to store this in a member variable but just get it when we need it like we do in the C++ sample?